project1-sps/script.js

```diff
@@ -1,4 +1,92 @@
+// initial mode & username variable
+var currentMode = "Enter Name";
+var userName = ""; // empty string, nothing stored
+
+// record user's Win Loss Draw score
+var userWinRecord = 0;
+var userLoseRecord = 0;
+var userDrawRecord = 0;
+
+// computerOption helper function
+var computerSPS = function (input) {
```

Comment:
Nice helper function! Maybe you can name it starting with a verb, something like `getComputerSPS`